src/switch_ivr_bridge.c

```diff
@@ -65,6 +67,30 @@ static void text_bridge_thread(switch_core_session_t *session, void *obj)
 
 	vh->up = 1;
 
+	if (!zstr(vh->session_a_uuid)) {
+		if (!(vh->session_a = switch_core_session_locate(vh->session_a_uuid))) {
+			vh->up = 0;
+			return;
+		}
+	} else {
+		vh->up = 0;
+		return;
```

Comment:
We allocate `text_framedata` and `text_buffer` above so when return here the allocation will leak